src/main/resources/hudson/tasks/junit/JUnitResultArchiver/config.jelly

```diff
@@ -25,22 +25,28 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
-	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-	<f:entry title="${%Test report XMLs}" field="testResults"
-		description="${%description('http://ant.apache.org/manual/Types/fileset.html')}">
-		<f:textbox />
-	</f:entry>
-	<f:entry field="keepLongStdio" title="">
-		<f:checkbox name="keepLongStdio" title="${%Retain long standard output/error}"/>
-	</f:entry>
-  <f:entry field="healthScaleFactor" title="${%Health report amplification factor}">
-    <f:number default="1.0" min="0" step="0.1" size="10"/>
-  </f:entry>
-	<j:invokeStatic var="testDataPublisherDescriptors"
-		className="hudson.tasks.junit.TestDataPublisher" method="all" />
-	<j:if test="${testDataPublisherDescriptors.size() > 0}">
-	<f:entry title="Additional test report features" field="testDataPublishers">
+    xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="${%Test report XMLs}" field="testResults"
+        description="${%description('http://ant.apache.org/manual/Types/fileset.html')}">
+        <f:textbox />
+    </f:entry>
+    <f:entry field="keepLongStdio" title="">
+        <f:checkbox name="keepLongStdio" title="${%Retain long standard output/error}"/>
+    </f:entry>
+    <f:entry field="healthScaleFactor" title="${%Health report amplification factor}">
+        <f:number default="1.0" min="0" step="0.1" size="10"/>
+    </f:entry>
+    <j:invokeStatic var="testDataPublisherDescriptors"
+        className="hudson.tasks.junit.TestDataPublisher" method="all" />
+    <j:if test="${testDataPublisherDescriptors.size() > 0}">
+    <f:entry title="Additional test report features" field="testDataPublishers">
             <f:repeatableHeteroProperty field="testDataPublishers" hasHeader="true" oneEach="true"/>
-	</f:entry>
-	</j:if>
+    </f:entry>
+    </j:if>
+    <f:advanced>
+        <f:entry title="${%Allow empty results}" field="allowEmptyResults">
+            <f:checkbox default="false" title="${%Do not fail the build on empty test results}"/>
+        </f:entry>
+    </f:advanced>
+    -->
```

Comment:
i think this line should be removed, the plugin version 1.10 got already released today
![someextrachars](https://cloud.githubusercontent.com/assets/11491375/11744368/0fcd26e2-a010-11e5-944f-c8c71d51e781.png)